### ISSUE TITLE
Misspelled --trivial in vg call options

### ISSUE THREAD

**Issue by cmarkello** [vgteam/vg#1416]
Discovered this typo when trying to generate gvcf-like output from `vg call`.

The option for making trivial calls in vg call is misspelled as --trival instead of the correct --trivial.

I believe the relevant typo can be found here:
https://github.com/vgteam/vg/blob/master/src/support_caller.hpp#L321

**Comment by glennhickey** [comment]
@cmarkello:

toil-vg will be fixed shortly (https://github.com/vgteam/toil-vg/pull/442)
to not crash in the event of double vcf entries.

Not sure if/when vg call --trival will be fixed to not produce dupes
#1419.  You can easily filter them yourself with bcftools filter, BUT
remember that this option was not intended to make GVCFs, especially not as
described here:
https://software.broadinstitute.org/gatk/documentation/article.php?id=4017

**Comment by cmarkello** [comment]
@glennhickey 

Yes it does look like in order to avoid making our own joint-genotyper, we would need to generate gvcfs in such a way as to match the reference confidence model that the gvcf format relies on. https://gatkforums.broadinstitute.org/gatk/discussion/4042/how-the-haplotypecallers-reference-confidence-model-works.
I don't fully understand what they mean by the use of a symbolic allele pair.
For the non-variant BLOCK records (as opposed to non-variant SITE records), they also have END tags in the INFO field indicating the end of the non-variant block.

The main hurdle I see is in trying to matchup with GATKs reference confidence model when making non-variant calls.

Either way, it seems that the most viable direction for cohort-based genotyping is to surject GAM files into BAM files and do GATK variant calling on these files. It's not efficient and basically bypasses the vg call/genotyper altogether. But it should work as a stop-gap for the time being.

**Comment by glennhickey** [comment]
I agree.  It seems your gam -> bam -> gatk pipeline was returning
nonsense.  I only suggested using vg call as an easy way to validate the
GAM, not as a means to feed directly into gatk.

**Comment by faithokamoto** [comment]
Well, as for the title, we no longer have any such `vg call --trivial` option, so it can't be misspelled :) For the other stuff, it looks handled too?
